tests/integ/modin/window/test_rolling.py

```diff
@@ -212,8 +235,11 @@ def test_rolling_window_unsupported():
             pd.Timestamp("20130101 09:00:06"),
         ],
     )
-    with pytest.raises(NotImplementedError):
-        snow_df.rolling(window="2s", min_periods=None).sum()
+    with pytest.raises(
+        NotImplementedError,
+        match="'center=True' is not implemented with str window for Rolling.sum",
```

Comment:
```suggestion
        match=re.escape("'center=True' is not implemented with str window for Rolling.sum"),
```

Reply:
I don't think this is necessary here?

Reply:
It makes a difference here because of the `.`. Compare these two context managers:

```python
import pytest
import re
with pytest.raises(NotImplementedError, match=r"Rolling.sum"):
    raise NotImplementedError("Rolling_sum")
with pytest.raises(NotImplementedError, match=re.escape(r"Rolling.sum")):
    raise NotImplementedError("Rolling_sum")
```

generally it's safer to use `re.escape` even if there's no special character in the match string, since someone might add a special character later.